core/eolearn/core/core_tasks.py

```diff
@@ -62,16 +62,23 @@ def __init__(self, path, filesystem=None, **kwargs):
             to 9 (highest compression).
         :type compress_level: int
         :param filesystem: An existing filesystem object. If not given it will be initialized according to the EOPatch
-            path
-        :type filesystem: fs.FS or None
+            path. If you intend to run this task in multiprocessing mode you shouldn't specify this parameter.
+        :type filesystem: fs.base.FS or None
```

Comment:
Just being pedantic, but I think the commonly used notation for this would be Optional[fs.base.Fs]. For exmaple if you used  MyPy typechecker and typing, you would write it like this. 
But no need to change this.

Reply:
Good point, but this would require a more systematic change everywhere. Besides, Sphinx recognizes the current way correctly, which can be seen from links in documentation.